semcore/data-table/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.30.0] - 2024-03-14
+
+### Changed
+
+- Pass `offset...` props to `ScrollArea` instead of custom styling in table.
```

Comment:
Actually I still can't get what bug you fixed or which feature you added 🤔

Reply:
Hope you will update the changelog note to a better one, other proposed changes are good 👍

Reply:
yep